src/routes/Events/components/EventsPage/EventsPage.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
-import DocumentTitle from 'react-document-title'
+import { Helmet } from 'react-helmet'
 
 import Event from '../Event'
 
 import styles from './EventsPage.scss'
 
 const EventsPage = ({ pastEvents, t }) => (
-  <DocumentTitle title={t('EventsPage.documentTitle')}>
+  <div>
```

Comment:
Here (and in other files as well) we don’t need to add an extra div :)

It’ll work fine under the `<div className={styles.events}>`